charters/DataStore/charter.md

```diff
@@ -0,0 +1,80 @@
+
+# Data Store
+
+
+## Description
+The Data Store is a scientific data sharing/publishing/distribution framework, providing file/bundle management on multiple clouds at petabyte-scale. It defines public APIs for storage, retrieval, and subscription to events that functions transparently across multiple cloud systems such as AWS and GCP.
+
+## Definitions
+**Bundle** A bundle is a list of related files along with some very basic metadata such as filenames.
+**DCP** The Data Coordination Platform is the name given to the entire system used to ingest, validate, store, analyzes, and make available the datga in the Human Cell Atlas project.
```

Comment:
Style nitpick, I think you need a double space at the end of the bundle definition otherwise md renderers don't recognise the new line

Reply:
thanks!